### PR TITLE
Unifying indentation: python-for-beginners/chapter 11-13

### LINKED ISSUE #37: Unifying indentation: python-for-beginners chapter 11-13
Unifying indentation to 4 spaces indent according to convention of #20, dealing python-for-beginners-... chapter 11-13

### PR DESCRIPTION
**Pull request by ChaeyeonSeo**
It resolves issue #37 .

### CODE REVIEW

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
         return(float(first_number) + float(second_number))
     elif operation.upper() =='SUBTRACT':
         return(float(first_number) - float(second_number))
     else:
         return('Invalid operation please specify ADD or SUBTRACT')
+
 # Test your function with the values 6,4, add 
-# Should return 10
-#
+# Should return 10#
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py]: What's the ending `#` means?

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 2]
```diff
@@ -5,22 +5,22 @@
...
-#
-
 def calculator(first_number, second_number, operation):
     if operation.upper() == 'ADD':
         return(float(first_number) + float(second_number))
     elif operation.upper() =='SUBTRACT':
         return(float(first_number) - float(second_number))
     else:
         return('Invalid operation please specify ADD or SUBTRACT')
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 3]
```diff
@@ -5,22 +5,22 @@
...
 # Test your function with the values 6,4, add 
-# Should return 10
-#
+# Should return 10#
 print('Adding 6 + 4 = ' + str(calculator(6,4,'add')))
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 4]
```diff
@@ -5,22 +5,22 @@
...
 # Test your function with the values 6,4, subtract 
 # Should return 2
 print('Subtracting 6 - 4 = ' + str(calculator(6,4,'subtract')))
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/print_time_function_different_messages.py [hunk 5]
```diff
@@ -8,7 +8,8 @@
 print() 
 
 for x in range(0,10):
-	print(x)
+    print(x)
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/print_time_function_different_messages.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/print_time_with_message_parameter.py [hunk 6]
```diff
@@ -9,12 +9,14 @@ def print_time(task_name):
     print()
 
 first_name = 'Susan'
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/print_time_with_message_parameter.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/print_time_with_message_parameter.py [hunk 7]
```diff
@@ -9,12 +9,14 @@ def print_time(task_name):
...
 # Call print_time() function to display message and current time
 # pass in name of task completed
 print_time('first name assigned')
 
 for x in range(0,10):
-	print(x)
+    print(x)
+
```
Comment (devquint) [inline python-for-beginners/13 - Functions/print_time_with_message_parameter.py]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 8]
```diff
@@ -5,22 +5,22 @@
 # operation: the word 'add' or 'subtract'
 # the function should return the result of the two numbers added or subtracted
 # based on the value passed in for the operator
-#
-
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py:9]: Do not handle with newline in this branch.

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 9]
```diff
@@ -5,22 +5,22 @@
 # operation: the word 'add' or 'subtract'
 # the function should return the result of the two numbers added or subtracted
 # based on the value passed in for the operator
-#
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py:8]: Do not handle with trim comments in this branch.

File: python-for-beginners/13 - Functions/code_challenge_solution.py [hunk 10]
```diff
@@ -5,22 +5,22 @@
...
     if operation.upper() == 'ADD':
         return(float(first_number) + float(second_number))
     elif operation.upper() =='SUBTRACT':
         return(float(first_number) - float(second_number))
     else:
         return('Invalid operation please specify ADD or SUBTRACT')
+
 # Test your function with the values 6,4, add 
-# Should return 10
-#
```
Comment (devquint) [inline python-for-beginners/13 - Functions/code_challenge_solution.py]: Do not handle with trim comments in this branch.

**Review by devquint — CHANGES_REQUESTED** [review]
1. **Check each comments of file/line**

2. 13 - `get_initials.py`, **should** be deal with line 12, better to deal with line 11-12

3. 13 - `get_initails_function.py`, better to deal with line 24-25

4. 13 - `getting_clever_with_functions_harder_to_read.py`, better to deal with line 17-20

**Review by devquint — APPROVED** [review]

**Review by Dorika0203 — APPROVED** [review]

**Review by Hyuntae-Jeong — APPROVED** [review]

**Review by jesusleejy — APPROVED** [review]

**Pull request merged**